### DISCUSSION TITLE
Some apps not updating?

### DISCUSSION THREAD

**Discussion by iGadget** [nextcloud/all-in-one#1871] [Questions] [answered]
Next to my Nextcloud AIO (automatic updates enabled), I'm also (still) running my old Nextcloud installation (now at v25 as well, installed years ago via HanssonIT script). On this old installation, I see regular update notices for apps, i.e. Passwords. For some time, I got the same notices on Nextcloud AIO, but a few week ago this changed, with the Passwords app showing this notification:

> [Passwords ends updates for Nextcloud 25](https://git.mdns.eu/nextcloud/passwords/wikis/Administrators/Notifications/Platform-Support-Notification)
Passwords 2023.1.20 is the last update for Nextcloud 25. Upgrade to Nextcloud 25 and PHP 8.1 (or PHP 7.4 for LSR) for future upgrades.

Since then, indeed the last update for Passwords was 2023.1.20. On my old Nextcloud 25 installation, Passwords is now at version 2023.1.23... 

What's going on here? And how do I debug this?

**Comment by szaimen** [comment]
Can you post the output of `sudo docker exec -it --user --www-data nextcloud-aio-nextcloud php occ app:list` here?

I suppose the notification is simply outdated...

**Reply by iGadget** [reply]
`docker exec -it --user www-data nextcloud-aio-nextcloud php occ app:list
Enabled:
  - activity: 2.17.0
  - admin_audit: 1.15.0
  - calendar: 4.2.2
  - circles: 25.0.0
  - cloud_federation_api: 1.8.0
  - co2clock: 0.1.3
  - collectives: 2.2.1
  - comments: 1.15.0
  - contacts: 5.0.3
  - contactsinteraction: 1.6.0
  - dashboard: 7.5.0
  - dav: 1.24.0
  - deck: 1.8.3
  - drawio: 2.1.0
  - external: 5.0.0
  - federatedfilesharing: 1.15.0
  - federation: 1.15.0
  - files: 1.20.1
  - files_external: 1.17.0
  - files_fulltextsearch: 25.0.0
  - files_pdfviewer: 2.6.0
  - files_rightclick: 1.4.0
  - files_sharing: 1.17.0
  - files_trashbin: 1.15.0
  - files_versions: 1.18.0
  - firstrunwizard: 2.14.0
  - fulltextsearch: 25.0.0
  - fulltextsearch_elasticsearch: 25.0.0
  - integration_google: 1.0.9
  - logreader: 2.10.0
  - lookup_server_connector: 1.13.0
  - maps: 0.2.4
  - nextcloud-aio: 0.3.0
  - nextcloud_announcements: 1.14.0
  - notes: 4.6.0
  - notifications: 2.13.1
  - notify_push: 0.5.2
  - oauth2: 1.13.0
  - password_policy: 1.15.0
  - passwords: 2023.1.20
  - photos: 2.0.1
  - privacy: 1.9.0
  - provisioning_api: 1.15.0
  - recognize: 3.3.6
  - recommendations: 1.4.0
  - related_resources: 1.0.3
  - richdocuments: 7.1.0
  - serverinfo: 1.15.0
  - settings: 1.7.0
  - sharebymail: 1.15.0
  - spreed: 15.0.3
  - support: 1.8.0
  - survey_client: 1.13.0
  - systemtags: 1.15.0
  - tasks: 0.14.5
  - text: 3.6.0
  - theming: 2.0.1
  - twofactor_backupcodes: 1.14.0
  - twofactor_nextcloud_notification: 3.5.0
  - twofactor_totp: 7.0.0
  - user_migration: 2.0.1
  - user_status: 1.5.0
  - viewer: 1.9.0
  - weather_status: 1.5.0
  - workflowengine: 2.7.0
Disabled:
  - apporder: 0.15.0
  - bruteforcesettings
  - encryption
  - suspicious_login
  - user_ldap`

**Reply by szaimen** [reply]
Can you check if there is an update available in the app store for the app?

**Reply by iGadget** [reply]
When I go to Apps, I see 'All apps are up-to-date.'

I suspected the server might not be able to check for updates (even though I assume Nextcloud would report this?), but the system can resolve an access hosts on the internet just fine. Anything else I can check?

**Comment by szaimen** [comment, marked as answer]
I rather think the reason is that the nextcloud container is currently using php 8.0. see https://github.com/nextcloud/all-in-one/issues/1782

**Reply by iGadget** [reply]
That's a reasonable explanation. I see my old Nextcloud server is indeed running PHP8.1 and that's on par with the [requirements](https://git.mdns.eu/nextcloud/passwords/-/wikis/Administrators/System-Requirements) of the latest version of Passwords. Hope #1782 can be fixed soon.
Thanks for your replies!
